main/acle.md

```diff
@@ -2830,6 +2826,50 @@ The following table lists the architectures feature mapping for AArch64
    | 580           | `FEAT_SME2`              | sme2          | ```ID_AA64PFR1_EL1.SMEver >= 0b0001```    |
    | 650           | `FEAT_MOPS`              | mops          | ```ID_AA64ISAR2_EL1.MOPS >= 0b0001```     |
 
+### Dependencies
+
+If a feature depends on another feature as defined by the table below then:
+
+* the depended-on feature *need not* be specified in the attribute,
+* the depended-on feature *may* be specified in the attribute.
+
+These dependencies are taken into account transitively when selecting the
+most appropriate version of a function (see section [Selection](#selection)).
```

Comment:
I don't agree with this - while it's unlikely to come up in practice, I think the selection algorithm would be simpler to understand if it were based solely upon the attribute strings and the feature priorities, without trying to detect and ignore any redundant dependencies.